src/lib/error-handler/index.ts

```diff
@@ -2,9 +2,46 @@ import { Response as ExResponse, Request as ExRequest, NextFunction } from 'expr
 import { ValidateError } from 'tsoa'
 
 import { logger } from '../logger'
-export interface ValidateErrorJSON {
-  message: 'Validation failed'
-  details: { [name: string]: unknown }
+
+/**
+ * this should reflect database tables
+ */
+type Entities = 'attachments' // add as needed
```

Comment:
I'm not sure I see the purpose of having another type to manage here, could we not pass the entity as an argument to the error? Then it could be `item: string`

Reply:
This will keep us a little bit more inline with the records. String means that it can be anything in this instance this works like enums but without the negative side.